tx.go

```diff
@@ -50,6 +63,7 @@ func (tx *Tx) init(db *DB) {
 		tx.pages = make(map[pgid]*page)
 		tx.meta.txid += txid(1)
 	}
+	tx.WriteToFlag = odirect
```

Comment:
Can you remove this default? I know it changes the default behavior but I think it improves the behavior for the vast majority of bolt users who have DBs smaller than RAM.